sfxclient/httpsink.go

```diff
@@ -36,16 +36,16 @@ const (
 	ClientVersion = "1.0"
 
 	// IngestEndpointV2 is the v2 version of the signalfx ingest endpoint
-	IngestEndpointV2 = "https://ingest.signalfx.com/v2/datapoint"
+	IngestEndpointV2 = "https://ingest.us0.signalfx.com/v2/datapoint"
 
 	// EventIngestEndpointV2 is the v2 version of the signalfx event endpoint
-	EventIngestEndpointV2 = "https://ingest.signalfx.com/v2/event"
+	EventIngestEndpointV2 = "https://ingest.us0.signalfx.com/v2/event"
 
 	// TraceIngestEndpointV1 is the v1 version of the signalfx trace endpoint
-	TraceIngestEndpointV1 = "https://ingest.signalfx.com/v1/trace"
+	TraceIngestEndpointV1 = "https://ingest.us0.signalfx.com/v1/trace"
```

Comment:
this should be at a minimum deprecated, at a maximum removed and major version incremented.

Reply:
Actually, i've thought more about this. Let's update this to v2/trace saying (leaving the deprecated flag you added) and describe why it's been changed to v2/trace... thoughts?

Reply:
updated let me know